### PR TITLE
[LTD-1028] Stop SLA for cases with terminal status.

### PR DESCRIPTION
**Pull request by alixedi**
If you read carefully, we are supposed to only stop the counter for `FINALISED`, `CLOSED`, `WITHDRAWN`.

I am going with `CaseStatusEnum.terminal_statuses()` because that seems like a more exhaustive list, which means it includes `FINALISED`, `CLOSED`, `WITHDRAWN` amongst others.

### CODE REVIEW

File: api/cases/tests/test_sla.py [hunk 1]
```diff
@@ -540,3 +543,35 @@ def test_sla_hours_does_not_appear_on_other_cases(self):
...
+        case = self.submit_application(application)
+        _set_submitted_at(case, HOUR_BEFORE_CUTOFF)
+
+        results = update_cases_sla.now()
+        case.refresh_from_db()
+        self.assertEqual(results, 1)
+        self.assertEqual(case.sla_days, 1)
+        self.assertEqual(case.sla_remaining_days, STANDARD_APPLICATION_TARGET_DAYS - 1)
+
+        for case_status in CaseStatusEnum.terminal_statuses():
```
Comment (alixedi) [inline api/cases/tests/test_sla.py:569]: It's a shame that I cannot parameterise a test that has to inherit from `DataTestClient` :(

File: api/cases/tests/test_sla.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 from unittest import mock
 from unittest.mock import patch
 
+import pytest
```
Comment (saruniitr) [inline api/cases/tests/test_sla.py:5]: this is not used

**Review by saruniitr — APPROVED** [review]

**Review by wkeeling — APPROVED** [review]

File: api/cases/tests/test_sla.py (hunk 2 above)
Comment (alixedi) [reply api/cases/tests/test_sla.py:5]: TY, Yes, wonder why the linter didn't complain.

File: api/cases/tests/test_sla.py (hunk 1 above)
Comment (eadpearce) [reply api/cases/tests/test_sla.py:566]: I noticed this the other day! Might be worth trying https://github.com/uktrade/lite-api/blob/49291035c015178cdd5bad9343409d4f48b50cef/api/common/tests/test_dates.py#L11

Comment (alixedi) [reply api/cases/tests/test_sla.py:566]: Awesome! TY @eadpearce. I tried it but it complicates some of the setup so sticking to the loop for now.

**Pull request merged**